packages/requires.txt

```diff
@@ -8,3 +8,4 @@ gunicorn>=20.1.0,<21
 pydantic[email]>=1.9.1,<2
 typing-extensions;python_version<'3.8'
 uvicorn[standard]>=0.17.6,<1
+fastapi-utils>=0.2.1
```

Comment:
This lib is no more maintened: https://pypi.org/project/fastapi-utils/ (last commit 2,5 years ago).
I think you should use background task instead: https://fastapi.tiangolo.com/tutorial/background-tasks/

Reply:
While it does seem like `https://pypi.org/project/fastapi-utils/` is not actively maintained, it does seem to be still alive with usage. In the end of the day it is[ a very thin wrapper over some asyncio / starlette functions](https://github.com/dmontagu/fastapi-utils/blob/master/fastapi_utils/tasks.py)
We can always fork it / take out just this function.

I do think using celery would be nice, but I think that there would be room to do it in a future upgrade.
As you say that would be a good time to review the policy polling mechanism as well.
(though truth be told, I really don't think anyone should be using polling for anything in prod  😆 - event-driven is the way to go)